# HG changeset patch
# User Rafael G. Martins <rafael@rafaelmartins.eng.br>
# Date 1344052271 10800
# Node ID c63455b2d73d91c64e74b0eaad4b621e966c58e0
# Parent  c0b3aee9771d15b265b673ae00f8a2fe6822e635
Fix tests, broken by a recent change on the Mercurial API.

--- a/blohg/testsuite.py
+++ b/blohg/testsuite.py
@@ -46,6 +46,7 @@
         self.app = create_app(self.repo_path, hgui=self.ui)
         create_repo(self.app)
         self.repo = hg.repository(self.ui, self.repo_path)
+        commands.add(self.ui, self.repo)
         self.app.hg.reload()
 
     def tearDown(self):
@@ -118,6 +119,7 @@
                                 'foo-bar.rst')
         with codecs.open(new_file, 'w', encoding='utf-8') as fp:
             fp.write(os.linesep.join(['Foo', '---', '', 'Bar']))
+        commands.add(self.ui, self.repo)
 
         self.app.hg.reload()
         with self.app.test_request_context():
@@ -154,6 +156,7 @@
 
         with codecs.open(new_file, 'w', encoding='utf-8') as fp:
             fp.write('foo')
+        commands.add(self.ui, self.repo)
 
         self.app.hg.reload()
         with self.app.test_request_context():
